echopype/convert/set_groups_ek60.py

```diff
@@ -268,7 +268,6 @@ def set_platform(self, NMEA_only=False) -> xr.Dataset:
                 )
             },
         )
-        ds = ds.chunk({"time1": DEFAULT_CHUNK_SIZE["ping_time"]})
 
         if not NMEA_only:
```

Comment:
I think you can remove this check for `NMEA_only` and the input argument -- this should be remnant of that datagram selection mechanism that was removed in v0.6.2 or right after (i.e. will be part of this release v0.6.3). Please double check but my quick search did not turn up with any usage of this variable.

Reply:
I double checked and I agree, I do not think this is ever used and is a remnant of the selection mechanism. I will go ahead and remove it.